openfl/component/aggregator/aggregator.py

```diff
@@ -854,15 +849,17 @@ def _end_of_round_check(self):
         if self._time_to_quit():
             self.logger.info('Experiment Completed. Cleaning up...')
         else:
+            self.round_number += 1
             self.logger.info(f'Starting round {self.round_number}...')
+            self.assigner.define_task_assignments_for_round(self.round_number)
```

Comment:
`define_task_assignments_for_round` is only implemented by the custom assigner; not the RandomGroupTaskAssigner or StaticGroupTaskAssigner

Reply:
Should we keep that assigners?

Reply:
Yes, those are needed for the TaskRunner API workflow